.github/workflows/integration_tests.yml

```
@@ -0,0 +1,51 @@
+name: Integration tests
```

```suggestion
name: Pod integration test
```
(It only runs 1 test, right?)

---

Yes, but it could require renaming later if we add some more, category shouldn't contain info that's about CocoaPods. That's why I have a more specific description in a job.